project-proposals/stage-2/osrd.md

```diff
@@ -0,0 +1,181 @@
+# Stage 2: Incubation
+
+## Health metrics
+
+### How many active contributors does the project have?
+
+https://github.com/OpenRailAssociation/osrd/graphs/contributors
+
+### How many different organizations are contributing to the project?
+
+- Almost all contribution comes from people working for SNCF Réseau.
+- Small contributions from single developers.
+
+### How much is the project dependent on individual contributors?
+
+*How high is the risk to a project should the most active people leave? This is called the [Contributor Absence Factor](https://chaoss.community/kb/metric-contributor-absence-factor/), sometimes also referred to as bus factor.*
+
+Computed with [bus-factor-explorer](https://github.com/JetBrains-Research/bus-factor-explorer).
+
+Repositories:
+- osrd (c98d3854cd1747a203f1ba6aa939901fbac72b75): **8**
+  - core: **2**
+  - editoast: **3**
+  - front: **4**
+  - gateway: **2**
+  - osrdyne: **1**
+- osrd-ui (3ef06474bae8b1c3d139a6b386c4c922eecfe8ef): **1**
+- osrd-chart (bbc948db28c417acb7714e3913a5eef148fed2e1): **1**
+
+-----
+
+Computed with [truck factor](https://github.com/HelgeCPH/truckfactor).
+
+The bus factor of https://github.com/OpenRailAssociation/osrd (68c3e1480c429a2cf8c7f8b7932769076761ac08) is: `4`
+with author(s): Eloi "eckter" Charpentier, Florian "flomonster" Amsallem, Victor "multun" Collod, Nicolas "nicolaswurtz" Wurtz
+
+The truck factor of https://github.com/OpenRailAssociation/osrd-ui (1f2fea5d6a0a153a5c02b5a96cd0a66627760b2b) is: 1
+with author(s): Chaka "kmer2016" NGAMENI NJINEH
+
+## Open governance
+
+### What is the release process of the project?
+
+- https://osrd.fr/en/docs/guides/release/
+
+### How are decisions made in the project?
+
+- https://github.com/OpenRailAssociation/osrd/blob/dev/GOVERNANCE.md
+- https://github.com/OpenRailAssociation/osrd/blob/dev/CONTRIBUTING.md
+
+### What is the business roadmap of the project?
+
+- https://osrd.fr/en/about/roadmap/
```

Comment:
Lightweight roadmap. Might be okay for this stage but should be more detailed for further stages.

> [!TIP]
> Could focus on the current year (2025)